lib/client/sse_client.dart

```diff
@@ -7,10 +7,17 @@ import 'dart:convert';
 import 'dart:html';
 
 import 'package:logging/logging.dart';
+import 'package:pool/pool.dart';
 import 'package:stream_channel/stream_channel.dart';
 
 import '../src/util/uuid.dart';
 
+/// Limit for the number of concurrent outgoing requests.
+///
+/// Chrome drops outgoing requests on the floor after some threshold. To prevent
+/// these errors we buffer outgoing requests with a pool.
+final _requestPool = Pool(1000);
```

Comment:
nit: it is probably all browsers not chrome. Comment on why 1000 is chosen for the pool size.